inst/examples/build-a-box/www/build-a-box.css

```diff
@@ -0,0 +1,69 @@
+/* FIXME: Remove after rstudio/htmltools#401 */
+.html-fill-container > .html-fill-item {
+  overflow: visible;
+}
```

Comment:
Has been merged to main, so could remove (but make sure to add version requirement & `Remotes`)